### PR TITLE
chore(demo): update getting started page with standalone example

### LINKED ISSUE #4766: 📚 - Update `Getting Started` page with standalone example
### Playground Link

//

### Description

Get error in console:

Assertion failed: EventPluginsModule must come after BrowserModule in imports

But, BrowserModule that doesn’t need to be explicitly imported when bootstrapping a Standalone Component.

### Angular version

16

### Taiga UI version

3.33

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by PblCbGomel**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [x] Documentation content changes

## What is the current behavior?

Closes #4766

### CODE REVIEW

File: projects/demo/src/modules/app/home/examples/main-standalone-optional.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import {HttpClientModule} from '@angular/common/http';
+import {provideAnimations} from '@angular/platform-browser/animations';
+import {provideRouter} from '@angular/router';
+import {TuiAlertModule, TuiDialogModule} from '@taiga-ui/core';
+import {TuiProprietaryRootModule} from '@taiga-ui/proprietary-core';
+import {routes} from './app/app.routes';
+
+bootstrapApplication(AppComponent, {
+  providers: [
+    importProvidersFrom(BrowserModule, HttpClientModule, TuiProprietaryRootModule, TuiDialogModule, TuiAlertModule),
```
Comment (waterplea) [inline projects/demo/src/modules/app/home/examples/main-standalone-optional.md:12]: `TuiRootModule`. Also I don't think first two are relevant.

File: projects/demo/src/modules/app/home/examples/app-standalone.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import {Component} from '@angular/core';
+import {TuiRootModule} from '@taiga-ui/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./my.component.scss', './my.component.less'],
+  standalone: true,
+  imports: [
+    TuiRootModule,
```
Comment (waterplea) [inline projects/demo/src/modules/app/home/examples/app-standalone.md:11]: `TuiRootComponent`

File: projects/demo/src/modules/app/home/examples/main-standalone.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+```ts
+import {TuiProprietaryRootModule} from '@taiga-ui/proprietary-core';
+import {BrowserModule} from '@angular/platform-browser';
+import {provideAnimations} from '@angular/platform-browser/animations';
+
+bootstrapApplication(AppComponent, {
+  providers: [
+    importProvidersFrom(BrowserModule),
```
Comment (waterplea) [inline projects/demo/src/modules/app/home/examples/main-standalone.md:8]: `TuiRootModule`

File: projects/demo/src/modules/app/home/examples/main-standalone.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    provideAnimations(), //extend BrowserAnimationModule
```
Comment (waterplea) [inline projects/demo/src/modules/app/home/examples/main-standalone.md:9]: I don't think this comment make sense.

File: projects/demo/src/modules/app/home/examples/main-standalone-optional.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    provideAnimations(), //extend BrowserAnimationModule
```
Comment (waterplea) [inline projects/demo/src/modules/app/home/examples/main-standalone-optional.md:13]: I don't think this comment make sense.

File: projects/demo/src/modules/app/home/examples/main-standalone-optional.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    provideRouter(routes), //for routing
```
Comment (waterplea) [inline projects/demo/src/modules/app/home/examples/main-standalone-optional.md:14]: This is irrelevant for the example

File: projects/demo/src/modules/app/home/examples/app-standalone.md (hunk 2 above)
Comment (vladimirpotekhin) [reply projects/demo/src/modules/app/home/examples/app-standalone.md:11]: no, it seems we need to import `TuiRootModule` here while `TuiRootComponent` is not standalone

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**